package.json

```diff
@@ -32,9 +35,6 @@
     "typescript": "^2.1.6"
   },
   "dependencies": {
-    "@types/bluebird": "^3.0.37",
-    "@types/lodash": "^4.14.52",
-    "@types/node": "^7.0.7",
```

Comment:
I'm not sure this is right. In a top-level component, you could move these into dev dependencies, but in a library that's not really the case I think. Downstream users of this module require these dependencies to be present to make sense of this module's type definitions, so they should indeed be actual dependencies.

If they're not dependencies, the exported types from this library that reference them are unusable unless the downstream component coincidentally already has these type definitions (which for these 3 examples is probably true, so this isn't so bad, but we should really do it 'right' anyway).

Paging @hedss, as this is something that I don't think our typescript guidelines cover, and we should clarify.